solvers/python-ADMM.py

```diff
@@ -12,7 +12,7 @@ class Solver(BaseSolver):
     name = 'ADMM analysis and synthesis'
 
     stopping_criterion = SufficientProgressCriterion(
```

Comment:
maybe instead of increasing the patience, try using the `sufficientDescentCriterion`?

Reply:
Yes! It is a lot faster!